sections/futures/PositionButtons/PositionButtons.tsx

```diff
@@ -65,10 +65,10 @@ const StyledPositionButton = styled(Button)<PositionButtonProps>`
 
 			${props.$isActive &&
 			css`
-				background: ${props.theme.colors.eliteTheme.button.active.hover.successBackground};
+				background: ${props.theme.colors.current.button.active.hover.successBackground};
```

Comment:
can we change `current` to `selectedTheme` that way it's a little bit more explicit to an outsider who may encounter this for the first time?

Reply:
Good catch. Will fix.